index.html

```diff
@@ -16,9 +16,9 @@
         <nav>
           <span id="menu_button"><img src="img/menu_icon.png" alt="menu" id="menu_icon"></span>
           <ul id="nav">
-            <li class="navigation">home</li>
-            <li class="navigation">about</li>
-            <li class="navigation">extra</li>
+            <li class="navigation"><img src="img/home.png" >home</li>
+            <li class="navigation"><img src="img/mglass.png">about</li>
+            <li class="navigation"><img src="img/hobbies.png">hobbies</li>
```

Comment:
You may want to use an icon font instead of adding images. They can be much easier to style than images, especially for menus like this. If you do not like using Icomoon, you could try using [Font Awesome](http://fontawesome.io/) instead.